controllers/keda/util/predicate.go

```diff
@@ -89,3 +92,14 @@ func (PausedPredicate) Update(e event.UpdateEvent) bool {
 
 	return newPausedValue != oldPausedValue
 }
+
+type HPAUpdatePredicate struct {
```

Comment:
```suggestion
type HPASpecChangedPredicate struct {
```
something like this is probably more descriptive

Reply:
sure, done the changes